Firmware/src/config/UIConfig.cpp

```diff
@@ -3,13 +3,13 @@
 void UIConfig::load() {
   Preferences prefs;
   prefs.begin(this->getConfigNamespace().c_str(), true);
-  this->accentColor = prefs.getUInt("accentColor", 0xFF0000);
+  this->accentColor = prefs.getInt("accentColor", 0xFF0000);
   prefs.end();
 }
 
 void UIConfig::save() {
   Preferences prefs;
   prefs.begin(this->getConfigNamespace().c_str(), false);
-  prefs.putUInt("accentColor", this->getAccentColor());
+  prefs.putInt("accentColor", this->getAccentColor());
```

Comment:
Given the conversations around fs handling I've left this as nvs for now. Once we have a decision I'm happy to move to JSON

Reply:
Let me try changing it back, I was using auto locally and noticed it was using int so changed the type

Reply:
Changed back to uint, it must have been something silly I did when I first made the changes that caused issues.